crates/astria-cli/src/cli/rollup.rs

```diff
@@ -0,0 +1,271 @@
+use std::str::FromStr;
+
+use clap::{
+    Args,
+    Subcommand,
+};
+use color_eyre::eyre;
+use serde::Serialize;
+
+/// Check if a string is a valid balance.
+fn is_valid_balance(s: &str) -> bool {
+    s.parse::<u128>().is_ok()
+}
+
+/// Manage your rollups
+#[derive(Debug, Subcommand)]
+pub enum Command {
+    /// Manage your rollup configs
+    Config {
+        #[clap(subcommand)]
+        command: ConfigCommand,
+    },
+    /// Manage your rollup deployments
+    Deployment {
+        #[clap(subcommand)]
+        command: DeploymentCommand,
+    },
+}
+
+/// Commands for managing rollup configs.
+#[derive(Debug, Subcommand)]
+pub enum ConfigCommand {
+    /// Create a new rollup config
+    Create(ConfigCreateArgs),
+    /// Edit a rollup config
+    Edit(ConfigEditArgs),
+    /// Delete a rollup config
+    Delete(ConfigDeleteArgs),
+}
+
+#[derive(Args, Debug, Serialize)]
+pub struct ConfigCreateArgs {
+    #[clap(long, env = "ROLLUP_USE_TTY")]
+    pub use_tty: bool,
+    #[clap(long, env = "ROLLUP_LOG_LEVEL")]
+    pub log_level: String,
+
+    // rollup config
+    #[clap(long = "rollup.name", env = "ROLLUP_NAME")]
+    /// The name of the rollup
+    pub name: String,
+    #[clap(long = "rollup.chain-id", env = "ROLLUP_CHAIN_ID", required = false)]
+    /// Optional. Will be derived from the rollup name if not provided
+    pub chain_id: Option<String>,
+    #[clap(long = "rollup.network-id", env = "ROLLUP_NETWORK_ID")]
+    pub network_id: u64,
+    #[clap(long = "rollup.skip-empty-blocks", env = "ROLLUP_SKIP_EMPTY_BLOCKS")]
+    pub skip_empty_blocks: bool,
+
+    #[clap(long = "rollup.genesis-accounts", env = "ROLLUP_GENESIS_ACCOUNTS", num_args = 1..)]
+    pub genesis_accounts: Vec<GenesisAccountArg>,
+
+    // sequencer config
+    #[clap(
+        long = "sequencer.initial-block-height",
+        env = "ROLLUP_SEQUENCER_INITIAL_BLOCK_HEIGHT",
+        required = false
+    )]
+    /// Optional. If not set, will be determined from the current block height of the sequencer
+    pub sequencer_initial_block_height: Option<u64>,
+    #[clap(long = "sequencer.websocket", env = "ROLLUP_SEQUENCER_WEBSOCKET")]
+    pub sequencer_websocket: String,
+    #[clap(long = "sequencer.rpc", env = "ROLLUP_SEQUENCER_RPC")]
+    pub sequencer_rpc: String,
+
+    // celestia config
+    #[clap(long = "celestia.full-node-url", env = "ROLLUP_CELESTIA_FULL_NODE_URL")]
+    pub celestia_full_node_url: String,
```

Comment:
After discussing with Jordan, I'm actually going to remove this arg for now. For this demo, we're not running with DA. I think setting a default here and keeping it in the config file could be confusing. It will be very easy to add back when we can run the full stack.